src/libsync/abstractnetworkjob.cpp

```diff
@@ -432,6 +382,8 @@ void AbstractNetworkJob::abort()
 {
     if (_reply) {
         _reply->abort();
+        _aborted = true;
+        // TODO: leak?
```

Comment:
Seems like the PR is still in a draft state?

Reply:
Was meant as a question to the reviewer but I answered it now.
Now I'm wondering whether the delete is correct or whether we should create a dummy reply with the error state...